src/cli/repl/commands/cfg.ts

```diff
@@ -15,7 +15,7 @@ async function controlflow(shell: RShell, remainingLine: string) {
 export const controlflowCommand: ReplCommand = {
 	description:  'Get mermaid code for the control-flow graph of R code, start with \'file://\' to indicate a file',
 	usageExample: ':controlflow',
-	aliases:      [ 'cfg' ],
+	aliases:      [ 'cf' ],
```

Comment:
```suggestion
	aliases:      [ 'cfg', 'cf' ],
```

There should be nothing preventing us from allowing both!
Especially because this would essentially be a breaking change without allowing the old alias.